<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <!-- Declare launch arguments -->
    <arg name="localization_topic" default="/ftd_master/localization_request" description="Localization request topic"/>

    <node
        pkg="autoware_bridge"
        exec="autoware_bridge_node"
        name="autoware_bridge"
        output="screen">
    </node>
</launch>
